### DISCUSSION TITLE
Compatibility question

### DISCUSSION THREAD

**Discussion by mcneds** [pybricks/support#820] [Q&A]
Is it possible to control a set of buwizz hubs and control +hubs with the pu hub running pybricks and controlling 1 motor but having brick controller 2 control another motor limited to only do steps of 90 degrees whilst still being activated in the app alongside the normally functioning buwizz controllers? The first motor controlled solely by control+ running pybricks should be fine but running the second hybrid thing for my gearbox im making seems like it may be incompatible.

**Comment by dlech** [comment]
Pybricks currently isn't compatible with Buwizz that I know of.
